Dockerfiles/manifests/cluster-agent/rbac-agent.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: datadog-agent
+rules:
+- apiGroups:  # Kubelet connectivity
```

Comment:
`# Kubelet connectivity:`

This is required by the agent to query the kubelet API.